server/src/main/java/server/database/system_information/SystemController.java

```diff
@@ -1,98 +1,80 @@
 package server.database.system_information;
 
-import com.google.gson.Gson;
-import com.mongodb.MongoException;
-import com.mongodb.client.FindIterable;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.MongoDatabase;
-import com.mongodb.util.JSON;
-import org.bson.BsonDocument;
 import org.bson.Document;
-import org.bson.types.ObjectId;
-import org.json.JSONArray;
-import org.json.JSONObject;
-
-import javax.print.Doc;
-import java.lang.reflect.Array;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-
-import static com.mongodb.client.model.Filters.eq;
+import org.bson.json.JsonMode;
+import org.bson.json.JsonWriterSettings;
 
 public class SystemController {
 
-    private final Gson gson;
-    private MongoDatabase database;
-
     private final MongoCollection<Document> abstractCollection;
     private final MongoCollection<Document> usersCollection;
     private final MongoCollection<Document> reviewGroupCollection;
 
+    private final JsonWriterSettings relaxed = JsonWriterSettings.builder().outputMode(JsonMode.RELAXED).build();
```

Comment:
Isn't this also deprecated? [Docs.](https://mongodb.github.io/mongo-java-driver/3.10/javadoc/org/bson/json/JsonWriterSettings.html)

Reply:
Constructor methods for JsonWriterSettings() are deprecated. It recommends using the .builder() function instead.